queue/abstract/queue_session.lua

```diff
@@ -0,0 +1,195 @@
+local log      = require('log')
+local fiber    = require('fiber')
+local uuid     = require('uuid')
+
+local util     = require('queue.util')
+local qc       = require('queue.compat')
+local num_type = qc.num_type
+local str_type = qc.str_type
+
+
+local queue_session = {
+    settings = {}
+}
+
+-- Sessions that don't have any active connections.
+local inactive_sessions = {}
+
+--- Create everything that's needed to work with "shared" sessions.
+local function identification_init()
+    local queue_session_ids = box.space._queue_session_ids
+    if queue_session_ids == nil then
+        queue_session_ids = box.schema.create_space('_queue_session_ids', {
+            temporary = true,
+            format = {
+                {name = 'box_session_id', type = num_type()},
```

Comment:
You call it 'connection_id' at least in the `get_uuid()` arguments. The module is abstracted out from box.session, so I would call it 'connection_id' uniformly.

The same for `box_sid` index name below.

Reply:
Accepted.